auth-web/src/stores/org.ts

```diff
@@ -831,6 +831,11 @@ export const useOrgStore = defineStore('org', () => {
     setCurrentOrganizationPADInfo(undefined)
   }
 
+  async function createOutstandingAccountPayment () {
+    const response = await PaymentService.createOutstandingAccountPayment(state.currentOrganization.id)
```

Comment:
This should be moved to the service? do we care about response.data? in the store? 
probably not..

Reply:
This doesn't do it with async, but I think we should make our services a bit more complicated (which should simplify stores):

https://github.com/bcgov/business-edit-ui/blob/2d76fb039f1dd6393574da807cbde30ef9a3e786/src/services/auth-services.ts#L60

Reply:
Makes sense, this would help the component logic to be more clean and less repetitive. I will be in this code soon doing EFT --> PAD screens probably. I will try to move the EFT calls out to the service when I do that.